README.md

```diff
@@ -92,7 +92,21 @@ if (!global.Intl) {
 
 _note: the locale data is required for the polyfill to function when using it in a browser environment, in the example above, the english (`en`) locale is being required along with the polyfill itself._
 
+If you're using [webpack 2], instead of `require.ensure(...)` use `System.import`:
+
+```javascript
+// ...
+
+    Promise.all([
+        System.import('intl'),
```

Comment:
Not experimental, it's in beta, people are starting to use it. I don't have strong opinions about it, I just thought this info might be useful. I'm not sure about why `System.import` is not in the spec, but that's what webpack 2 uses… Feel free to close 😉